### PR TITLE
Support rails 5.2

### PR DESCRIPTION
**Pull request by novotnyjakub**
Apart from adding build support for Rails `5.2.x` it also updates the build versions of Ruby to the latest patches available in the `deliveroo/multiruby` Docker image.

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -18,7 +18,14 @@ appraise 'rails-5' do
 end
 
 appraise 'rails-5-1' do
-  gem 'rails', '~> 5.1'
+  gem 'rails', '~> 5.1.0'
+  gem 'sqlite3'
```
Comment (pedrocunha) [inline Appraisals]: why does it need the news gems?

Comment (novotnyjakub) [reply Appraisals]: Well I made sure that this appraisal is only for `5.1.x` and not for `5.x`. The `sqlite` I'm not sure, to be honest, but it's in all of the other appraisal groups as well. I think the tests were failing without it.

**Comment by pedrocunha** [comment]
This looks. How do you feel about adding 2.5 since we are here?

**Comment by novotnyjakub** [comment]
Btw. @pedrocunha I can't merge this, because I don't have permission. So please feel free to merge this PR so that we can release it. Whenever you have some time, of course. Thanks!

**Pull request merged**